lib/commons/dom/is-focusable.js

```diff
@@ -9,22 +9,33 @@ import { getNodeFromTree } from '../../core/utils';
  * @memberof axe.commons.dom
  * @instance
  * @param {HTMLElement} el The HTMLElement
+ * @param {object} options Optional. If set to {programatically: false}, the function will return true
  * @return {Boolean} The element's focusability status
  */
-export default function isFocusable(el) {
+export default function isFocusable(el, options = { programmatically: false }) {
```

Comment:
No need to set the default

```suggestion
export default function isFocusable(el, options = {}) {
```